tests/integration/test_logcollector/test_location_custom_sockets/test_location_custom_sockets.py

```diff
@@ -0,0 +1,289 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+from os import path, unlink
+from socket import AF_UNIX, SHUT_RDWR, SOCK_STREAM, SOCK_DGRAM, socket
+from tempfile import gettempdir
+
+import pytest
+
+from wazuh_testing import global_parameters
+from wazuh_testing.logcollector import (GENERIC_CALLBACK_ERROR_COMMAND_MONITORING, callback_analyzing_file,
+                                        callback_socket_connected, callback_socket_offline,
+                                        get_next_stats, get_data_sending_stats)
```

Comment:
This is growing too much. Instead of this, you can import the module as follows:
```python
import wazuh_testing.logcollector as logcollector
```
or
```python
import wazuh_testing.logcollector as lg
```
And then, use the functions from the module as follows:
```python
lg.GENERIC_CALLBACK_ERROR_....
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/pull/1338/commits/058ef4e5b590a99da16710e8a6780e64516906fa